mu4e/mu4e-main.el

```diff
@@ -198,7 +198,7 @@ clicked."
 
 (defun mu4e~main-view ()
   "Create the mu4e main-view, and switch to it."
-  (if (eq mu4e-split-view 'single-window)
+  (if mu4e-hide-main-view-buffer
       (if (buffer-live-p (mu4e-get-headers-buffer))
```

Comment:
Seems like this breaks the existing behavior, no?

Reply:
Indeed, with `mu4e-split-view` set to `single-window`, the default behavior is now to display the main view buffer. To recover the former default behavior of displaying the main menu instead, `mu4e-hide-main-view-buffer` must be set to `t`. For other values of `mu4e-split-view`, the default behavoir is unchanged.

I'm not sure how to make the main view / main menu choice independent of the split view mode without breaking any default behavior. One possibility would be to make the default value of `mu4e-hide-main-view-buffer` dependent on the value of `mu4e-split-view`, but that wouldn't work for users who set `mu4e-split-view` after loading mu4e. Or perhaps we could only offer the choice for single-window mode (e.g. adding a variable `mu4e-single-window-hide-main-view` defaulting to `t`) and assume users of other split-view modes want the main view buffer.